scipy/stats/stats.py

```diff
@@ -6306,6 +6352,116 @@ def ttest_rel(a, b, axis=0, nan_policy='propagate', alternative="two-sided"):
     return Ttest_relResult(t, prob)
 
 
+def ttest_trimmed(a, b, axis=0, equal_var=False, nan_policy='propagate',
```

Comment:
Perhaps save this code somewhere but let's remove it from here. I think it's looking like this will integrate with `ttest_ind` nicely.